gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/management/JdbcPortalNavigationItemRepository.java

```diff
@@ -64,26 +64,49 @@ public List<PortalNavigationItem> findAllByOrganizationIdAndEnvironmentId(String
     }
 
     @Override
-    public List<PortalNavigationItem> findAllByAreaAndOrganizationIdAndEnvironmentId(
-        PortalNavigationItem.Area area,
-        String organizationId,
-        String environmentId
-    ) throws TechnicalException {
-        log.debug(
-            "JdbcPortalNavigationItemRepository.findAllByAreaAndOrganizationIdAndEnvironmentId({}, {}, {})",
-            area,
-            organizationId,
-            environmentId
-        );
+    public List<PortalNavigationItem> findAllByParentIdAndEnvironmentId(String parentId, String environmentId) throws TechnicalException {
+        log.debug("JdbcPortalNavigationItemRepository.findAllByParentIdAndEnvironmentId({}, {})", parentId, environmentId);
         try {
-            final String sql = getOrm().getSelectAllSql() + " where area = ? and organization_id = ? and environment_id = ?";
-            return jdbcTemplate.query(sql, getOrm().getRowMapper(), area.name(), organizationId, environmentId);
+            final String sql =
+                "select id, organization_id, environment_id, title, type, area, parent_id, \"order\", configuration from " +
```

Comment:
getOrm().getSelectAllSql().

The main benefit of using this method, is that it automatically escape reserved word, using the right way depending on the DBMS.
What you do with double-quotes does not necessary work for SQLServer or MySQL.
See https://github.com/gravitee-io/gravitee-api-management/blob/aff62c371ab3527449f16fc0435a1564e581aa5d/gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/java/io/gravitee/repository/jdbc/common/AbstractJdbcRepositoryConfiguration.java#L181